packages/mirinae/src/data-display/field-title/PFieldTitle.vue

```diff
@@ -1,37 +1,47 @@
 <template>
-    <label class="p-field-title" v-on="$listeners">
-        <slot>
-            {{ label }}
-        </slot>
-    </label>
+    <span class="p-field-title"
+          v-on="$listeners"
+    >
+        <label>
+            <slot>
+                {{ props.label }}
+            </slot>
+        </label>
+        <span v-if="props.description"
+              class="description"
+        >
+            {{ props.description }}
+        </span>
+    </span>
```

Comment:
Question:
Is there any specific reason why the span tag is wrapping the label tag?
As far as I know, the label tag is typically used with input elements, so if the purpose of the label is for specific styling, what do you think about applying the styles using CSS?

Reply:
No there is no reason. I think it just span tag is proper in this case!